external/vcpkg_x64-windows_response_file.txt

```diff
@@ -0,0 +1,5 @@
+openssl
```

Comment:
In the latest restructuring of vcpkg, I have cleaned it up and organized things in a better way. We will have to get rid of these files but can delete it after your PR is merged.